### ISSUE TITLE
Something that should be normal in 2.5.3 becomes error in 2.6

### ISSUE THREAD

**Issue by LowScarlet** [SkriptLang/Skript#4225]
### Skript/Server Version

```
[06:04:31 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[06:04:31 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[06:04:31 INFO]: [Skript] Server Version: git-Purpur-1292 (MC: 1.17.1)
[06:04:31 INFO]: [Skript] Skript Version: 2.6-beta2
[06:04:31 INFO]: [Skript] Installed Skript Addons:
[06:04:31 INFO]: [Skript]  - Skellett v1.9.11 (https://forums.skunity.com/resources/skellett.24/)
[06:04:31 INFO]: [Skript]  - skript-gui v1.3-alpha1 (https://github.com/APickledWalrus/skript-gui)
[06:04:31 INFO]: [Skript]  - skript-yaml v1.4
[06:04:31 INFO]: [Skript]  - HolographicDisplaysAddon v1.0.0b
[06:04:31 INFO]: [Skript]  - skRayFall v1.9.26 (https://sk.rayfall.net/)
[06:04:31 INFO]: [Skript]  - SkBee v1.11.2 (https://github.com/ShaneBeee/SkBee)
[06:04:31 INFO]: [Skript]  - skript-placeholders v1.5.1 (https://github.com/APickledWalrus/skript-placeholders)
[06:04:31 INFO]: [Skript]  - AdvancementsAddon v1.0.0
[06:04:31 INFO]: [Skript]  - DiSky v2.2 (https://disky.itsthesky.info/)
[06:04:31 INFO]: [Skript]  - skDragon v0.17
[06:04:31 INFO]: [Skript]  - skUtilities v0.9.2 (https://tim740.github.io/)
[06:04:31 INFO]: [Skript]  - skript-reflect v2.2.2-dev (https://github.com/TPGamesNL/skript-reflect)
[06:04:31 INFO]: [Skript]  - Reqn v1.2.4
[06:04:31 INFO]: [Skript]  - SkQuery v4.1.5
[06:04:31 INFO]: [Skript]  - Skript-Paper v1.3.0 (https://github.com/ShaneBeee/Skript-Paper)
```

### Bug Description

This
```hs
function test_0(aaa: text) :: number:
      return 2021
function test_1():
      set {_v} to "%test_0(""aaaa"")%"
      broadcast {_v}
```
Getting error "can't understand"

### Expected Behavior

It should not error and give a broadcast of the variable {_v}

### Steps to Reproduce

```hs
function test_0(aaa: text) :: number:
      return 2021
function test_1():
      set {_v} to "%test_0(""aaaa"")%"
      broadcast {_v}
on load:
     test_1()
```

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by LowScarlet** [comment]
This will be fixed by put the value "aaa" into the variable
like
```hs
set {_aaaa} to "aaaa"
set {_v} to "%test_0({_aaaa})%"

**Comment by UsainSrht** [comment]
It's weird but in 2.6 double quotes in parentheses and percent signs aren't allowed i think.
`set {_v} to "%test_0("aaaa")%"` <= the fix

**Comment by AyhamAl-Ali** [comment]
> It's weird but in 2.6 double quotes in parentheses and percent signs aren't allowed i think.
> `set {_v} to "%test_0("aaaa")%"` <= the fix

They have been changed in Skript-beta1
> Fix quote parsing. This change allows singular double quotes to be used in variables and (expressions within strings), instead of doubled double quotes (#3762, closes #590)
https://github.com/SkriptLang/Skript/releases/tag/2.6-alpha1
